sig-contributor-experience/role-handbooks/comm-mgmt-lead.md

```diff
@@ -0,0 +1,41 @@
+# (Community Management Subproject Lead) OR (Lead Moderator) Role Description
+
+The lead moderator is the glue between our many communication platforms geared 
+towards upstream development, the volunteers that moderate them, and project 
+leadership, specifically SIG-Contributor-Experience.  
+
+### [Title] MUST:
+- Ensure onboarding and offboarding for all platform [moderators]
+- Provide data about the platforms, moderators (no personal info), incidents, 
+and more to Code of Conduct Committee for annual transparency report or ad hoc 
+for [OWNERs of kubernetes/community] for various community reporting
+- Ensure all moderator positions are full and if not, create a recruitment plan
+- Establish check-ins with current moderators
+- Stay on top of all platform changes and what that means for our processes, 
+moderation, and administration
+- Keep all moderation guidelines and relevant documentation and up to date
+- Act as liasion between moderation team and Code of Conduct Committee
+
+### [Title] MAY:
+- Delegate the above responsonibilities to the shadow or another moderator 
+- Suggest automation or process improvements to a communication platform
+- Remove another moderators permissions 
+
+### Term:
```

Comment:
Any limits on terms/re-elections?

Reply:
im thinking the role is intended to turn over after a year as its necessary to have a shadow; however, if there is no shadow for whatever reason at that time and no suitable replacement, another 1 year? how does that sound?

Reply:
Unilateral removing of someone's permissions makes me nervous. Can we have a mandatory +1 from another moderator?

Reply:
+1 to @jdumars -- removal of elevated permissions should require a +1 from another mod or lead (or shadow?)